taskcluster/ci/version-bump/kind.yml

```diff
@@ -21,6 +21,8 @@ only-for-build-types:
     - focus-release
 
 job-template:
+    attributes:
+        shipping_phase: ship     # APKs are sent to testing tracks
```

Comment:
Does this comment belong here?

Reply:
Oops, that's a bad copy'n'paste. Thanks for spotting it. Fixed!